### DISCUSSION TITLE
Unable to do a numbered list with directives on each item.

### DISCUSSION THREAD

**Discussion by jtpavlock** [pradyunsg/furo#190] [Potential Issue] [answered]
The [solution to this stackoverflow question](https://stackoverflow.com/questions/29354305/sphinx-resume-list-numbering-after-a-note-section) doesn't work because indented directives in furo are automatically treated as blockquoted-text, and everything becomes italicized.

**Comment by jtpavlock** [comment, marked as answer]
Actually, I had my directive tabbed (4 spaces) which was causing it to become blockquoted. If I reset the indentation to only 3 spaces, it works.

**Reply by pradyunsg** [reply]
Furo bothers to stylise blockquotes -- the element becomes a blockquote when using other themes as well; they just don't bother with stylising them which means these mistakes fall through. For those other themes, those block quotes are invisible to all readers except those with screen readers, who just have a worse experience due to stray block quotes everywhere.

**Reply by jtpavlock** [reply]
Thanks for explanation and the great theme! I just switched and am loving it so far!
